From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:33 +0100
Subject: dont-build-xiu

---
 OMakefile           |    2 --
 ocaml/xiu/OMakefile |    4 ----
 2 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/OMakefile b/OMakefile
index 6004c0e..ec63fe5 100644
--- a/OMakefile
+++ b/OMakefile
@@ -115,8 +115,6 @@ OCAML_PHASE3_XEN = \
 	allxenops \
 	ocaml/xenguest/xenguest \
 	ocaml/xenguest/dumpcore \
-	ocaml/xiu/libxenctrl_xiu.so \
-	ocaml/xiu/xiu \
 	ocaml/xapi/fakeguestagent \
 	ocaml/xapi/quicktestbin \
 	ocaml/xapi/sparse_dd \
diff --git a/ocaml/xiu/OMakefile b/ocaml/xiu/OMakefile
index dd2a183..f49a016 100644
--- a/ocaml/xiu/OMakefile
+++ b/ocaml/xiu/OMakefile
@@ -18,13 +18,9 @@ clean:
 .PHONY: install
 install:
 	mkdir -p $(LIBEXEC)
-	$(IPROG) xiu $(LIBEXEC)/
 	mkdir -p $(BASE_PATH)/lib
-	$(IDATA) libxenctrl_xiu.so $(BASE_PATH)/lib
 
 .PHONY: sdk-install
 sdk-install:
 	mkdir -p $(LIBEXEC)
-	$(IPROG) xiu $(LIBEXEC)/
 	mkdir -p $(BASE_PATH)/lib
-	$(IDATA) libxenctrl_xiu.so $(BASE_PATH)/lib
-- 
