From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:42 +0100
Subject: fix-base-path-script

---
 OMakefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/OMakefile b/OMakefile
index ec63fe5..2ba59fc 100644
--- a/OMakefile
+++ b/OMakefile
@@ -50,7 +50,7 @@ XEN_OCAML_CLIBS=
 CFLAGS+=-I$(shell ocamlc -where)
 
 DESTDIR=$(getenv DESTDIR, $(ROOT)/dist/staging)
-RAW_BASE_PATH=$(shell scripts/base-path scripts/xapi.conf)
+RAW_BASE_PATH=$(shell bash scripts/base-path scripts/xapi.conf)
 BASE_PATH=$(DESTDIR)$(RAW_BASE_PATH)
 BIN_DIR=$(RAW_BASE_PATH)/bin
 BIN_PATH=$(DESTDIR)$(BIN_DIR)
-- 
