From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:44 +0100
Subject: debian-qemu-location

---
 scripts/qemu-dm-wrapper |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/qemu-dm-wrapper b/scripts/qemu-dm-wrapper
index 583ad4e..2b6a453 100755
--- a/scripts/qemu-dm-wrapper
+++ b/scripts/qemu-dm-wrapper
@@ -79,7 +79,7 @@ def main(argv):
 	import os
 
 	qemu_env = os.environ
-	qemu_dm = '/usr/lib/xen/bin/qemu-dm'
+	qemu_dm = '/usr/lib/xen-4.1/bin/qemu-dm'
 	domid = int(argv[1])
 	qemu_args = ['qemu-dm-%d'%domid] + argv[2:]
 
-- 
