From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:50 +0100
Subject: [PATCH] fix-rewrite-management-interface

---
 scripts/rewrite-management-interface |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/rewrite-management-interface b/scripts/rewrite-management-interface
index 252a363..b88cbb7 100755
--- a/scripts/rewrite-management-interface
+++ b/scripts/rewrite-management-interface
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Copyright (c) Citrix Systems 2008. All rights reserved.
 #
-- 
