From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:54 +0100
Subject: vif-hot-unplug

---
 scripts/vif |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/vif b/scripts/vif
index d09fad8..bee842a 100755
--- a/scripts/vif
+++ b/scripts/vif
@@ -255,7 +255,7 @@ add)
     fi
     ;;
 
-remove)
+remove | offline)
     if [ "${TYPE}" = "vif" ] ;then
         call_hook_script $DOMID "${ACTION}"
 		# Unclear whether this is necessary, since netback also does it:
-- 
