From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:51 +0100
Subject: [PATCH] pam

---
 scripts/pam.d-xapi |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/pam.d-xapi b/scripts/pam.d-xapi
index f18960b..485bf18 100644
--- a/scripts/pam.d-xapi
+++ b/scripts/pam.d-xapi
@@ -1,4 +1,4 @@
 #%PAM-1.0
-auth       include     system-auth
-account    include     system-auth
-password   include     system-auth
+auth       include     common-auth
+account    include     common-auth
+password   include     common-auth
-- 
