From: Mike McClurg <mike.mcclurg@citrix.com>
Date: Sun, 11 Dec 2011 15:13:34 +0000
Subject: fix-with-vdi

---
 scripts/with-vdi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/with-vdi b/scripts/with-vdi
index ae4e878..daa6ffb 100755
--- a/scripts/with-vdi
+++ b/scripts/with-vdi
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-function usage()
+function usage
 {
   echo
   echo Usage:
-- 
